src/katgpucbf/fgpu/engine.py

```diff
@@ -1581,21 +1590,13 @@ def comma_string_to_float(comma_string: str) -> tuple[float, float]:
             phase, phase_rate = comma_string_to_float(phase_str)
 
             delay_samples = delay * self.adc_sample_rate
-            # For compatibility with MeerKAT, the phase given is the net change in
-            # phase for the centre frequency, including delay, and we need to
-            # compensate for the effect of the delay at that frequency. The centre
-            # frequency is 4 samples per cycle, so each sample of delay reduces
-            # phase by pi/2 radians.
-            delay_phase_correction = 0.5 * np.pi * delay_samples
-            phase += delay_phase_correction
-            phase_rate_correction = 0.5 * np.pi * delay_rate
```

Comment:
Was this bit not correct? Was it replaced by what is now on lines 785 - 798?

Reply:
Replaced by line 797 for the coarse delay and line 261 in postproc.mako for the fine delay.